editor/editor_properties.cpp

```diff
@@ -4174,9 +4174,12 @@ struct EditorPropertyRangeHint {
 	bool radians = false;
 };
 
-static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const String &p_hint_text, double p_default_step) {
+static EditorPropertyRangeHint _parse_range_hint(PropertyHint p_hint, const String &p_hint_text, double p_default_step, bool is_int = false) {
 	EditorPropertyRangeHint hint;
 	hint.step = p_default_step;
+	if (is_int) {
+		hint.hide_slider = false; // Always show slider for ints, unless specified in hint range.
```

Comment:
There is no logic in the spin box to properly time the undo/redo as was done in #69797.

I remember there was a problem with this, that every time a button is pressed in FPS mode, an undo redo is recorded. There are places where the buttons should be hidden, so if you want them to show by default, we need to add a negative hint like `hidebutton`.

Reply:
AFAIK this PR still respects the `hide_slider` hint, which will hide the buttons.

Reply:
Ah I see, I will test it.